README.md

```diff
@@ -45,3 +45,23 @@ n := 4000;; numberChops := 8;; q := 5;;
 A := RandomMat(n, n, GF(q));;
 MeasureContention(numberChops, q, A, true);
 ```
+
+## Contact
+
+Please submit bug reports, suggestions for improvements and patches via
+the [issue tracker](https://github.com/gap-system/PackageMaker/issues).
+
+You can also contact me directly via [email](max@quendi.de).
+
+## License
+
+`PackageMaker` is free software you can redistribute it and/or modify it
```

Comment:
Do I understand correctly that this is the licence of the PackageMaker that we used? Shouldn't we add a licence for our project now that we made it a `real` GAP package?

Reply:
Hm, I must have lsot a commit somewhere. You're right, it should say `GaussPar`.